### PR TITLE
Fixes 3536 Use actual value of author name instead of static string

### LINKED ISSUE #3536: Replace static summary in custom author name with actual value
**Summary:** 

Continuing the discussion from [thread](https://github.com/commons-app/apps-android-commons/pull/3521#issuecomment-600058029)

Currently, a static summary is set which is somewhat repetition of the above switchpref

76859067-db11a980-6858-11ea-9898-1b8e9bf5d382

According to me, it would be better to show **Not Set** or the actual value.
Screenshot_1584450599

Screenshot_1584450593

**Would you like to work on the issue?**
Yes, I already have a PR for this

### PR DESCRIPTION
**Pull request by madhurgupta10**
**Description**
Fixes #3536 

**Tests performed**

Tested betaDebug on Google Pixel 2 with API level 29.

**Screenshots showing what changed**
Screenshot_1584450593
Screenshot_1584450599

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -390,7 +390,6 @@
   <string name="preference_author_name_toggle">Use custom author name</string>
   <string name="preference_author_name_toggle_summary">Use a custom author name instead of your username while uploading photos</string>
   <string name="preference_author_name">Custom author name</string>
-  <string name="preference_author_name_summary">The custom author name to use instead of your username in uploads</string>
```
Comment (macgills) [inline app/src/main/res/values/strings.xml]: not removed for every other locale, run a lint inspection and they should show up as unused resources to delete

Comment (madhurgupta10) [reply app/src/main/res/values/strings.xml]: @macgills I think the **translate-wiki** automatically removes them from other locals

Comment (madhurgupta10) [reply app/src/main/res/values/strings.xml]: Pinging @misaochan for getting more information on this.

Comment (sivaraam) [reply app/src/main/res/values/strings.xml]: I believe you're right. Translate wiki handles this by itself. Ref: [FAQ](https://translatewiki.net/wiki/FAQ#Do_I_have_to_do_anything_special_when_deleting_messages?)

Comment (macgills) [reply app/src/main/res/values/strings.xml]: Coolio

**Pull request merged**